AE_PunktBeschriftung/include/representations/Solution.hpp

```diff
@@ -1,32 +1,58 @@
-#pragma once
+#ifndef LABEL_MAP_SOLUTION_HPP
+#define LABEL_MAP_SOLUTION_HPP
 
-#include <representations/Instance.hpp>
-#include <representations/Point.hpp>
 
-#include <map>
+#include "Instance.hpp"
 
 class Solution {
-
-private:
-
-    const Instance& instance;
-
-    std::map<int, Point::Rectangle> placements;
-
-    friend std::ostream& operator<<(std::ostream &ostream, const Solution &solution);
+protected:
+    const Instance instance;
```

Comment:
This really should be a reference, we do not want to copy the instance (if we are doing it this way, we do not need Solutions at all and could have kept everything in the instance)

Reply:
I had that in mind, probably a typo :D